pages/en/lb3/Discovering-models-from-relational-databases.md

```diff
@@ -15,11 +15,15 @@ summary:
 LoopBack makes it simple to create models from an existing relational database.
 This process is called _discovery_ and is supported by the following connectors:
 
-* [MySQL connector](MySQL-connector.html)
-* [PostgreSQL connector](PostgreSQL-connector.html)
-* [Oracle connector](Oracle-connector.html)
+* [Cassandra](Cassandra-connector.html)
+* [MySQL](MySQL-connector.html)
+* [Oracle](Oracle-connector.html)
+* [PostgreSQL](PostgreSQL-connector.html)
 * [SAP HANA](https://www.npmjs.org/package/loopback-connector-saphana) - Not officially supported; see [Community connectors](Community-connectors.html).
-* [SQL Server connector](SQL-Server-connector.html)
+* [SQL Server](SQL-Server-connector.html)
+* [IBM DB2](DB2-connector.html)
+* [IBM DashDB](DashDB.html)
+* [IBM DB2 for z/OS](DB2-for-z-OS.html)
```

Comment:
Same comment about moving SAP HANA at the bottom of the list for better organization